spaceship/lib/spaceship/client.rb

```diff
@@ -484,8 +484,6 @@ def send_shared_login_request(user, password)
       # Now we know if the login is successful or if we need to do 2 factor
 
       case response.status
-      when 403
```

Comment:
403 not handled any more at all?

Reply:
https://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses The "old" 403 case handles 401 as well, but is not as precise, because it can mean different things. If you try it out in the browser or check the spaceship logs, you will see that there are actual more helpful responses from Apple. I moved the logic down to lines 497+ to further split this into different and more helpful response messages

Reply:
Well, important is what Apple returns: if they return a 403 in some cases, before that was explicitly handled with these lines. Now it is not handled by the status code any more.